Solutions/Azure Active Directory/Analytic Rules/SigninBruteForce-AzurePortal.yaml

```diff
@@ -76,5 +91,5 @@ entityMappings:
     fieldMappings:
```

Comment:
Can we add AAD User ID as an entity as well please.
Also is it possible to get a device entity included in this?

Reply:
Added the AAD User ID as an entity also, Kindly please checkout